docs/topics/pagination.txt

```diff
@@ -2,15 +2,68 @@
 Pagination
 ==========
 
+Django provides high-level and low-level ways to help you manage paginated data
+-- that is, data that's split across several pages, with "Previous/Next" links.
+
+Paginating a `ListView`
+=======================
+
+:class:`django.views.generic.list.ListView` provides a built-in way to
+paginate the displayed list. You can use it by adding to ``paginate_by``
+attribute to your view class, like so::
+
+    from django.views.generic import ListView
+    from books.models import Publisher
+
+    class PublisherList(ListView):
+        paginate_by = 2
+        model = Publisher
+
+Assuming you were able to view the full list of publishers before this change,
+the example above already works! The only thing your users will be missing is
```

Comment:
I'm not sure about this sentence. There's no explicit example of what it looked like before the change, and surely the code as specified will work whether or not it was correct before it was modified. I'd probably just say something simpler like "This is the only change necessary to the view code." Personally I prefer it without an exclamation mark, though I don't think it's actually wrong.